documentation/docs/developers/building-and-deploying-to-ios.md

```diff
@@ -33,17 +33,17 @@ If the deployment has not previously been built for iOS, follow these steps
 config.ios.app_id = "international.idems.my-example-app"
 config.ios.app_name = "My Example App"
 ```
+1. An up-to-date `GoogleService-Info.plist` file for the deployment's registered Firebase app is required to be populated to `ios/App/App/GoogleService-Info.plist`. This can be downloaded from the Firebase console.
 2. Run `yarn workflow ios configure` to generate the relevant iOS build files from the templated files using the values specified in the deployment config.
 3. Generate iOS assets (app icon and splash screen), using the [capacitor-assets](https://github.com/ionic-team/capacitor-assets) package. Alternatively, the assets can be generated in [XCode directly](https://developer.apple.com/documentation/xcode/configuring-your-app-icon) after step 6.
 4. Run `yarn build`
 5. Run `npx cap sync ios`
 6. Run `npx cap open ios` to open the project in XCode.
-7. Open the Firebase console and fetch the relevant `GoogleService-Info.plist` file for the deployment's registered app. Copy the file to `ios/App/App/GoogleService-Info.plist`.
-8. In order to sign the app, in XCode navigate to `App > Targets > App Signing & capabilities`. Set the value for the development team, using "Add account" if you have not previously configured XCode with a development team. Ensure that the option "Automatically manage signing" is checked.
-9. Now build the app to a target simulator to test functionality.
-10. When ready to upload the app to App Store Connect, set the target as "Any iOS device (arm64)", and go to `Product > Archive`.
+7. In order to sign the app, in XCode navigate to `App > Targets > App Signing & capabilities`. Set the value for the development team, using "Add account" if you have not previously configured XCode with a development team. Ensure that the option "Automatically manage signing" is checked.
```

Comment:
Thanks for updating this docs, still handy to have as we slowly migrate more over to CI